src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,180 @@
+package org.fundacionjala.coding.jorge;
+
+/**
+ *
+ */
+public final class FizzBuzz {
+    private static final int NUMBER_5 = 5;
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_10 = 10;
+
+    /**
+     * The first part.
+     *
+     * @param rango int[]
+     * @return Object[]
+     */
+    public Object[] fizzBuzz(final int[] rango) {
+        Object[] arrayObj = new Object[rango[1] - rango[0] + 1];
+        arrayObj = fill(rango[0], arrayObj);
+        arrayObj = stage0(arrayObj);
+        arrayObj = stage1(arrayObj);
+        arrayObj = stage2(arrayObj);
+        return arrayObj.clone();
+    }
+
+    /**
+     * The second part.
+     *
+     * @param rango int[].
+     * @return arrayObj2 Object[].
+     */
+    public Object[] fizzBuzz2(final int[] rango) {
+        Object[] fizz2 = new Object[]{};
+        fizz2 = fizzBuzz(rango);
+        fizz2 = stage3(fizz2);
+        fizz2 = stage4(fizz2);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment; "fizz2" already holds the assigned value along all execution paths. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4165)